docs/UserGuide.md

```diff
@@ -169,23 +165,38 @@ Examples:
 - `list` followed by `delete 2` deletes the 2nd contact in the address book.
 - `find Betsy` followed by `delete 1` deletes the 1st contact in the results of the `find` command.
 
-### Clearing all entries : `clear`
+#### Clearing all entries : `clear`
 
 Clears all entries from the address book.
 
 Format: `clear`
 
-### Exiting the program : `exit`
+#### Exiting the program : `exit`
 
 Exits the program.
 
 Format: `exit`
 
-### Saving the data
+### Other features
+
+#### Sending an email to a specific contact
+
+Opens the desktop mail application to directly send an email to a specific contact.
```

Comment:
I think there should be a standardisation for our other features section as well. 

Maybe we can do something like 
Description: ....
How to use:....

Wdyt?

Reply:
Sure, I think we can just have an opening paragraph to describe the feature, then there should be a How to use section like you said.